workshop/serializers.py

```diff
@@ -480,9 +481,9 @@ def save(self, **kwargs):
     class Meta:
         model = Workshop
         fields = (
-            'id', 'title', 'description', 'date', 'time', 'location', 'latitude',
-            'longitude', 'audience', 'contacts', 'image_url', 'tags',
-            'link')
+            'id', 'title', 'description', 'is_workshop', 'date', 'time',
```

Comment:
Since this serializer is only used for creating "workshops", maybe we should not take the "is_workshop" field in the json body. While creating workshops, this can be done directly: `Workshop.objects.create(..., is_workshop=True)` @Vikhyath08

Reply:
So then do you propose creating a separate endpoint for creation of an event?

Reply:
I think you already made two separate endpoints: `/entities/{id}/workshops/create/` and `/clubs/{id}/workshops/create/`

Reply:
Maybe I got confused, I thought an entity will always have an event, and a club will always have a workshop. Isn't it like this?
So, an entity can have both events and workshops, same with the clubs?
If yes, then it's fine.

Reply:
As far as I've understood, both entities and clubs can create both workshops and events. This is why I've given an option to check whether the created "workshop" is a workshop or an event. Maybe @nishantkr18 can provide a bit more clarity on this?

Reply:
> Maybe I misunderstood it, I thought an entity will always have an event, and a club will always have a workshop. Isn't it like this? So, an entity can have both events and workshops, same with the clubs?
> If yes, then it's fine.

This what I believe is happening so that clubs like CoPs can conduct events like hackathons, while also allowing entities like the ecell or SCS to conduct their workshops...

Reply:
Yup, that makes sense. Great work!